react/gulpfile.js

```diff
@@ -68,6 +68,31 @@ function resolvePath(sourcePath, currentFile, opts) {
     if (check.test(sourcePath)) {
       const matches = check.exec(sourcePath);
       resolvedPath = `${rootPath}${matches[1]}`;
+      // ES Modules need to list out the extension on the end of the path,
```

Comment:
Can you explain what this is doing? I looked at index.js in `dist/` before and after and don't see anything different with this change:
https://github.com/massgov/mayflower/pull/1098/files#diff-1ed6f9d93c1ab4ea1481408cc9aab6daR71-R95''